templates/profiles/change_password.html

```diff
@@ -0,0 +1,20 @@
+{% extends "base.html" %}
+{% load i18n %}
+{% load markdownify %}
+{% load courses_extra %}
+{% load bootstrap_tags %}
+
+{% block content %}
+<form method="post">
+    {% csrf_token %}
+    <div class="py-5 container bg-light">
+            <table style="margin-right: auto; margin-left: auto">
+                {{ form.as_table }}
+            </table>
+            <button type="submit" style="margin: 0 auto; display: block;margin-top: 10px">Uložiť zmeny</button>
```

Comment:
Please do not use inline style. This would cause a lot of issues down the road... If anyone would style the page, inline style would cause a lot of problems, use bootstrap, of if it is not adequace you can use your own css class (that ca be easily overwritten in CSS.

But this should be sufficient (or something along this lines):

```
<div class="col-3 mx-auto py-4 text-center">
        <button class="btn btn-lg btn-primary btn-block" type="submit">{% translate "Log in" %}</button>
      </div>
```

Reply:
yap, i'm sorry once again, i will do my best, not really familiar with bootstrap but at least there is an opportunity to dive into a bit....we prefer bootstrap over css itself, right?